src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int ballsLimit = 3;
```

Comment:
it should be constant and placed as class-level
`private static final int NUMBER_OF_BALLS = 3; `

Reply:
Are you sure that we can use 'privat static' modifiers in this case??? 
It will lead to an error 'illegal start of expression'

Reply:
ok